tests/tested-files.json

```diff
@@ -0,0 +1,265 @@
+{
+    "benchmarks_path" : "~/formela/sv-benchmarks/c/",
```

Comment:
This should be configurable and it should be noted somewhere that you need to clone some external repository.